ckanext/datastore/tests/test_search.py

```diff
@@ -634,6 +634,18 @@ def test_search_is_unsuccessful_when_called_with_invalid_fields(self):
         assert res_dict['success'] is False
         assert res_dict['error'].get('fields') is not None, res_dict['error']
 
+    def test_search_without_total(self):
+        data = {'resource_id': self.data['resource_id'],
+                'include_total': False}
+        postparams = '%s=1' % json.dumps(data)
+        auth = {'Authorization': str(self.normal_user.apikey)}
+        res = self.app.post('/api/action/datastore_search', params=postparams,
+                            extra_environ=auth)
+        res_dict = json.loads(res.body)
+        assert res_dict['success'] is True
+        result = res_dict['result']
+        assert 'total' not in result
+
```

Comment:
Would you add this to the class at the top instead of with the legacy tests and like those tests use `helpers.call_action` to skip the request processing bits.